Sources/SwiftParserDiagnostics/ParserDiagnosticMessages.swift

```diff
@@ -706,7 +721,7 @@ public struct RemoveRedundantFixIt: ParserFixIt {
 public struct RemoveNodesFixIt: ParserFixIt {
   public let nodesToRemove: [Syntax]
 
-  init(_ nodesToRemove: [some SyntaxProtocol]) {
+  init(_ nodesToRemove: [any SyntaxProtocol]) {
```

Comment:
`any SyntaxProtocol` requires us to create an existential, which is quite costly. I would prefer to keep it as-is and instead pass `Syntax` for any kind of heterogeneous list.